src/app/views/main/main.component.scss

```diff
@@ -31,9 +31,21 @@
             position: absolute;
             right: 0;
             margin: 1rem;
+            display: flex;
+            width: 15rem;
+            justify-content: space-between;
 
             .search-criteria {
               font-weight: 500;
+              line-height: 1.6rem;
+            }
+
+            .download-button {
+              cursor: pointer;
+
+              .download-icon {
+                color: #757575;
```

Comment:
Should use a variable from the global color file (look at other scss to figure it out)